modules/user_accounts/php/my_preferences.class.inc

```diff
@@ -417,20 +417,9 @@ class My_Preferences extends \NDB_Form
             array('UID' => $this->identifier)
         );
 
-        // If we are using PHP5.5+ and entry Password_has in the DB is not null
-        // use method password_verify to check if the password changed
-        if (function_exists('password_verify')
-            && !is_null($passwords['Password_hash'])
-        ) {
+        if (!is_null($passwords['Password_hash'])) {
             return !password_verify($newPassword, $passwords['Password_hash']);
-        } elseif (!function_exists('password_verify')
-            && !is_null($passwords['Password_hash'])
-        ) {
-            // If PHP version < 5.5, use old MD5Unsalt method to check for
-            // password change
-            return !\User::MD5Unsalt($newPassword, $passwords['Password_hash']);
```

Comment:
User::MD5Unsalt isn't declared in the user class anymore, so this would have failed anyways. We don't support < 5.5 anymore, so I took out the check for the password_verify function.